libcloudforensics/providers/aws/forensics.py

```diff
@@ -117,12 +120,16 @@ def CreateVolumeCopy(zone,
             kms_key_id, destination_account_id)
         # Create a copy of the initial snapshot and encrypts it with the
         # shared key
-        temporary_snapshot = snapshot.Copy(kms_key_id=kms_key_id)
-        # Delete the initial snapshot
-        snapshot.Delete()
-        snapshot = temporary_snapshot
+        snapshot = snapshot.Copy(kms_key_id=kms_key_id, delete=True)
       snapshot.ShareWithAWSAccount(destination_account_id)
 
+    if dst_zone:
+      # Assign the new zone to the destination account and assign it to the
+      # snapshot so that it can copy it
+      destination_account.__init__(dst_zone, aws_profile=dst_account)
```

Comment:
I are we calling `__init__` here and not through the normal python constructor (`destination_account()`) ?
If this is functionality we need to call to set things up, then please add a function to the destination_account's class to do so.

Reply:
Hmmm I don't see why I did that :D I reverted to default constructor.